iac/terraform/aws/api_gw.tf

```diff
@@ -81,11 +81,11 @@ resource "aws_lambda_permission" "lambda_invoke_permission" {
 
 #resource "aws_api_gateway_domain_name" "api_gw_domain_name" {
 #  domain_name     = "${var.environment}.${var.bloodconnect_domain}"
-#}
-#
-#resource "aws_api_gateway_base_path_mapping" "api_mapping" {
+# }
+
```

Comment:
if we are don't need commented code, let's remove this as we can always check git history

Reply:
I thought to keep it as it's not written by me.